helm-charts/splunk-otel-collector/values.schema.json

```diff
@@ -1153,6 +1153,15 @@
       "description": "Apply for k8s cluster with windows worker node.",
       "type": "boolean"
     },
+    "securityContextConstraints": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "create": {
+          "type": "boolean"
+        }
+      }
+    },
     "securityContextConstraintsOverwrite": {
```

Comment:
With the proposed changes, we should migrate securityContextConstraintsOverwrite to be securityContextConstraints.overwrite.
- Add a helper function to accept both securityContextConstraintsOverwrite and securityContextConstraints.overwrite.
- Add a note to UPGRADING.md about how using to prefer using securityContextConstraints.overwrite going forward.

Reply:
Does this need to be part of this PR? I have a hard time finding the capacity to contribute, so I am thinking if it can be its own PR.